src/test/resources/application-test.yml

```diff
@@ -45,3 +45,22 @@ cancel:
   queue:
     purge-frequency-time: 100ms
     purge-total-time: 5s
+
+hmpps.sqs:
+  provider: localstack
+  queues:
+    migrationvisits:
+      visibilityTimeout: 1
```

Comment:
overrridden defaults to timeout after 1 second for messages that aren't processed and to only retry once

Reply:
Does this mean that with real queues that have a visibility timeout of 30 seconds we won't get the message on the DLQ for 30 seconds? Mind you, guess that doesn't really matter.